packages/config/src/android/Icon.ts

```diff
@@ -1,24 +1,152 @@
+import { resolve } from 'path';
+import Jimp from 'jimp';
+import fs from 'fs-extra';
 import { ExpoConfig } from '../Config.types';
-import { addWarningAndroid } from '../WarningAggregator';
+import * as Colors from './Colors';
+
+type dpiMap = { [dpistring: string]: { folderName: string; scale: number } };
```

Comment:
I would go with sth like this here: https://www.staging-typescript.org/play?#code/MYewdgzgLgBAIgBQJIGUoCcCWYDmEYC8MA2gOQC2AJgA6akA0MpAFjXY6QB6u0NOfc2fLgJ50AujACG+UJCgBuAFBQAntQCm8ZGiy5CMNZpAAzbagzY8xMAFdyAIw3pxyo1rYBZKdQMAlDVB0SgAeRAs9HEYAbxgTEAAbSmcAOSlyDQAuGGhIhRzgKQSsmDtHZxgAXwA+BSA
```suggestion
const DPIStrings = ['mdpi', 'hdpi', 'xhdpi', 'xxhdpi', 'xxxhdpi'] as const;
type DPIString = typeof DPIStrings[number];
type dpiMap = Record<DPIString, { folderName: string; scale: number }>;
```
💅